website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -1,203 +1,84 @@
-
-:::note 
-To ensure automatic backups before updates, enable the `WATCHTOWER_LIFECYCLE_HOOKS` environment variable in the [docker-compose.yaml](https://bit.ly/3h08WqB) file on the Watchtower instance.
-:::
-
-If you have updated your Appsmith instance and face any issues. You can rollback the changes and [restore the Appsmith instance](/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) from a backup archive. This configuration runs an Appsmith instance and a Watchtower instance to keep Appsmith automatically up-to-date. The installation is now configured to stay up-to-date automatically.
-
----
-
-## Setup with Docker run
-
-<Tabs groupId="editions">
-   <TabItem label="Community Edition" value="community">
-
-This command downloads the image and starts Appsmith:
-
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ce
-```
-
-   </TabItem>
-   <TabItem label="Business Edition" value="business">
-
-This command downloads the image and starts Appsmith:
-
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ee
-```
-
-   </TabItem>
-</Tabs>
-
-### Updating Appsmith
-
-<Tabs groupId="editions">
-   <TabItem label="Community Edition" value="community">
-
-To update Appsmith manually, go to the root directory of the installation and run the following commands:
-
-```bash
-docker rmi appsmith/appsmith-ce -f
-docker pull appsmith/appsmith-ce
-docker rm -f appsmith
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" appsmith/appsmith-ce
-```
-
-   </TabItem>
-   <TabItem label="Business Edition" value="business">
-
-To update Appsmith manually, go to the root directory of the installation and run the following commands:
+2. To start the Docker container, use the following command. You may have to run the command with `sudo` if you don't have permission to execute `docker-compose`. The below command downloads the Docker image if not present locally.
 
 ```bash
-docker rmi appsmith/appsmith-ee -f
-docker pull appsmith/appsmith-ee
-docker rm -f appsmith
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" appsmith/appsmith-ee
+docker-compose up -d
 ```
 
-   </TabItem>
-</Tabs>
-
-Once the download is complete, the server should be up soon. You can verify that it's working correctly in the [Docker logs](#docker-logs).
-
-## Docker logs
-
-Once the Appsmith container is ready, the message "Appsmith is Running!" is displayed in the Docker logs.
+The server deploys Appsmith at startup, and you can access it at [http://localhost](http://localhost). You can also check the message "Appsmith is Running!" displayed in docker logs by using `docker logs -f appsmith` command.
 
 ![Appsmith is running message](/img/InstallationGuides__Docker__AppsmithRunningMessage.png)
```

Comment:
i think we should avoid this screenshot

Reply:
Yes, I had removed it. But then put it back after our sync review call in the morning.